### PR TITLE
React-Task100

### PR DESCRIPTION
**Pull request by mishalfirdous**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import AboutSection from "./aboutSection";
+import ExperienceSection from "./experienceSection";
+import Sidebar from "./sidebar";
+import './style.css'
+
+
+
+function App() {
+  return (
+    <div>
```
Comment (huzaifah0x00) [inline src/App.jsx:12]: Please setup and use [prettier ](https://prettier.io/) for formatting the code.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+/*footer section  */
+.FooterSection{
+        background-color: white;
+        height: 70px;
+        padding: 20px;
+        display: flex;
+        justify-content: left;
+        border: 2px solid lightgray;
+    }
```
Comment (huzaifah0x00) [inline src/style.css]: keep the naming convention consistent for all your css classes.
Above you have kebab-case (.about-section), here you have PascalCase ( .FooterSection ).
its better to use the same case everywhere, kebab-case or camelCase is most commonly used with css. use one of these.

**Review by huzaifah0x00 — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (mishalfirdous) [reply src/App.jsx:12]: fixed!

File: src/style.css (hunk 2 above)
Comment (mishalfirdous) [reply src/style.css]: fixed!

**Review by huzaifah0x00 — APPROVED** [review]

**Pull request merged**